ps_checkout.php

```diff
@@ -22,6 +22,7 @@
 use PrestaShop\Module\PrestashopCheckout\PayPal\Intent;
 use PrestaShop\Module\PrestashopCheckout\PayPal\Mode;
 use PrestaShop\Module\PrestashopCheckout\PayPal\PayPalConfiguration;
+use PrestaShop\ModuleLibServiceContainer\DependencyInjection\ServiceContainer;
```

Comment:
I missed this part in the last PRs, using `use [...]` won't be compliant with PrestaShop 1.6 (the class will be detected as invalid)

Reply:
```suggestion
```